app/views/posts/index.html.erb

```diff
@@ -26,6 +26,7 @@
           <div class="commentsandlikes">
             <h5 class="sign">Comments:</h5>
```

Comment:
- [ ] Good job by adding screenshots of the n + 1 problem in PR description. However, after checking in all your code, I can't find anywhere you added code lines to fix that. Kindly ensure that you fix that as it is for the project requirements. And for that, you will ensure that to NOT use the `bullet` gem. Example in the posts#index actions students should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`

Reply:
hi i never used that in that file i do not get what you mean

Reply:
oh you were referring to all of them, understood.